protractor/e2e/integration/blog-page.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { BlogPo } from '../support/blog.po';
+import { SecondPo } from '../support/second.po';
+
+describe("tests for blog page", () => {
+    const blogPage = new BlogPo();
+    const secondPage = new SecondPo();
+
+    beforeEach(async () => {
+        await secondPage.navigateTo();
+        await secondPage.headerMenuBlog.click();
+    });
+
+    it("check blog articles and pagination", async () => {
+        const expectedArticlesNumber = 20;
+        expect(await blogPage.getCurrentUrl()).toContain('blog.ng-book.com');
+        expect(await blogPage.postsSection.isDisplayed()).toBeTruthy();
+        expect(await blogPage.article.count()).toEqual(expectedArticlesNumber);
+        expect(await blogPage.articleTitle.count()).toEqual(expectedArticlesNumber);
+        expect(await blogPage.articleDescription.count()).toEqual(expectedArticlesNumber);
+        expect(await blogPage.articleReadMore.count()).toEqual(expectedArticlesNumber);
+        await blogPage.scrollToElement(blogPage.pagingBlock);
+        expect(await blogPage.pagingLast.isDisplayed()).toBeTruthy();
+        expect(await blogPage.pagingNext.isDisplayed()).toBeTruthy();
+        expect(await blogPage.pagingFirst.isDisplayed()).toBeTruthy();
+        expect(await blogPage.pagingSecond.isDisplayed()).toBeTruthy();
+        await blogPage.pagingSecond.click();
+        await blogPage.waitForVisible(blogPage.postsSection);
+        expect(await blogPage.getCurrentUrl()).toContain('page/2/');
+        expect(await blogPage.article.count()).toBeGreaterThan(0);
+    });
+});
```

Comment:
It's advisable to divide all this logic to 2 it blocks: the first one for checking "Navigate to blog and check articles, pages navigation"
And the second one: "Navigate to blog and check articles, pages navigation"